pkg/provider/provider.go

```diff
@@ -576,3 +576,12 @@ func createNodes(nodes []corev1.Node) map[string]Node {
 
 	return wrapperNodes
 }
+func GetBaremetalNodes(env *TestEnvironment) []Node {
+	var baremetalNodes []Node
+	for _, node := range env.Nodes {
+		if strings.HasPrefix(node.Data.Spec.ProviderID, "baremetalhost://") {
```

Comment:
I feel like this would be a good built-in function for our `Node` objects similar to the other ones [here](https://github.com/test-network-function/cnf-certification-test/blob/main/pkg/provider/nodes.go).

Then you could write `if node.IsBaremetal`.

Reply:
david asked to be like this and i think its better becuse of the skip if empty